frank-flow/src/frontend/src/app/shared/services/flow-namespace.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { CurrentFileService } from './current-file.service';
+import { File } from '../../shared/models/file.model';
+import { FlowStructure } from '../models/flow-structure.model';
+import { MonacoEditorComponent } from 'src/app/editor/monaco-editor/monaco-editor.component';
+import { FlowStructureService } from './flow-structure.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FlowNamespaceService {
+  public monacoEditorComponent?: MonacoEditorComponent;
+  private currentFile!: File;
+
+  constructor(
+    private currentFileService: CurrentFileService,
+    private flowStructureService: FlowStructureService
+  ) {
+    this.getflowStructure();
+  }
+
+  getflowStructure(): void {
```

Comment:
You changed changed the name, but also changed the contend. The old name would be more fitting now, don't you think?